src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -193,27 +219,31 @@ describe('526 helpers', () => {
     });
   });
 
-  describe('transformDisabilities', () => {
-    const rawDisability = initialData.ratedDisabilities[1];
-    const formattedDisability = Object.assign(
-      { disabilityActionType: 'INCREASE' },
-      rawDisability,
-    );
-    it('should create a list of disabilities with disabilityActionType set to INCREASE', () => {
-      expect(transformDisabilities([rawDisability])).to.deep.equal([
-        formattedDisability,
-      ]);
-    });
-    it('should return an empty array when given undefined input', () => {
-      expect(transformDisabilities(undefined)).to.deep.equal([]);
-    });
-    it('should remove ineligible disabilities', () => {
-      const ineligibleDisability = _.set(
-        'decisionCode',
-        SERVICE_CONNECTION_TYPES.notServiceConnected,
-        rawDisability,
+  describe('setActionTypes', () => {
+    const formData = maximalData.data;
+
+    it('should set disabilityActionType for each disability properly', () => {
+      const formattedDisabilities = setActionTypes(formData).ratedDisabilities;
+
+      expect(formattedDisabilities.length).to.equal(
```

Comment:
Also possible: [`.to.have.lengthOf(...)`](https://www.chaijs.com/api/bdd/#method_lengthof)

I'm wondering if we should start making it a habit to use more of these semantic assertions. I think it might just affect the error output, but :man_shrugging: 

It's fine by me either way, at any rate.